Bad timezone implementation in datetimes

**Describe the bug**
Unit tests regarding datetime actions are failing ever since local timezone changed from CEST to CET, caused by hardcoded values of `+0200` hours in UTC offset when the provided sign `UtcTime` field is incorrect (showing same time as in `TrueDate` yet `+0000` in `UtcTime`). Now failing tests show a 1-hour difference in assertion outputs.

**To Reproduce**
Steps to reproduce the behavior:

1. Run unit tests
2. Wait for them to fail

**Expected behavior**
All timezone-sensitive tests should pass.

**Logs**

```
=========================== short test summary info ============================
FAILED tests/test_woffu_api_client.py::TestWoffuAPISummaryDiary::test_get_summary_report_slot_without_motive_different_timezones - AssertionError: 7.394225 != 8.394225 within 7 places (1.0000000000000009 difference)
!!!!!!!!!!!!!!!!!!!!!!!!!! stopping after 1 failures !!!!!!!!!!!!!!!!!!!!!!!!!!!
======================== 1 failed, 133 passed in 2.42s =========================
```

**Desktop (please complete the following information):**

- OS: Any
- Version: 0.1.5

**Additional context**
It'd be wise to discard any use of the `UtcTime` as it brings more confusion than anything else. Working with `trueDate` field might be enough for our use-cases, treating it as an already timezone-aware date, being the local timezone the one applied.

--- a/src/woffu_client/woffu_api_client.py
+++ b/src/woffu_client/woffu_api_client.py
@@ -440,29 +440,10 @@ diarysummaries/{diary_summary_id}/workday/slots/self",
         current_time = datetime.now(tz=self._localzone)
 
         for sign in signs_in_day:
-            # running_clock = sign['SignIn']
-            # sign_date = sign['TrueDate']
-            # # Split UtcTime and keep only the offset
-            # utc_offset = f"{sign['UtcTime'].split(' ')[1].zfill(3)}00"
 
             running_clock = sign.get("SignIn", False)
             sign_date = sign.get("TrueDate", None)
-            utc_time = sign.get("UtcTime", "")
-            try:
-                # Split UtcTime and keep only the offset (format '+HHMM')
-                utc_offset = f"{utc_time.split(' ')[1].zfill(3)}00"
-            except (IndexError, AttributeError):
-                utc_offset = UTC_OFFSET  # fallback to UTC
-
-            # WORKAROUND: Woffu stores incorrect UTC information
-            # when signing in from the Webapp, showing same local date
-            # on 'TrueDate' and 'UtcTime' but with no UTC offset (+0).
-            # - For the time being, we'll be using our local timezone
-            #   for any sign that comes with +0 in the UtcTime as we
-            #   suspect Woffu DB is running on a server with the same
-            #   local timezone as us.
-            if utc_offset == UTC_OFFSET:
-                utc_offset = current_time.strftime("%z")
+            utc_offset = current_time.strftime("%z")
 
             sign_date_timezoned = f"{sign_date}{utc_offset}"
 
@@ -529,15 +510,6 @@ diarysummaries/{diary_summary_id}/workday/slots/self",
 
         # Send sign request
         logger.info("Sending sign request...")
-        # # Get the current datetime with local timezone
-        # current_time = datetime.now(tz=self._localzone)
-
-        # # Get the actual time offset in minutes
-        # utc_offset = current_time.utcoffset()
-        # if utc_offset:
-        #     timezone_offset = -(int(utc_offset.total_seconds() / 60))
-        # else:
-        #     timezone_offset = 0
 
         return self.post(
             url=f"https://{self._domain}/api/svc/signs/signs",
@@ -660,40 +632,18 @@ Using `in`/`out` keys...",
 
     def _calculate_hours_from_in_out(self, slot: dict) -> float:
         """Calculate hours from 'in' and 'out' keys of a slot."""
-        in_utc_parts = slot["in"]["utcTime"].split(" ")
-        out_utc_parts = slot["out"]["utcTime"].split(" ")
-        if len(in_utc_parts) < 2 or len(out_utc_parts) < 2:
-            logger.warning(
-                f"Skipping slot with invalid UTC times: "
-                f"in='{slot['in']['utcTime']}'"
-                f"out='{slot['out']['utcTime']}'",
-            )
-            return 0.0
+        in_dt = self._parse_datetime(slot["in"]["trueDate"])
+        out_dt = self._parse_datetime(slot["out"]["trueDate"])
 
-        in_dt = self._parse_datetime(slot["in"]["trueDate"], in_utc_parts[1])
-        out_dt = self._parse_datetime(
-            slot["out"]["trueDate"], out_utc_parts[1],
-        )
         return (out_dt - in_dt).total_seconds() / 3600
 
-    def _parse_datetime(self, date_str: str, utc_offset: str) -> datetime:
+    def _parse_datetime(self, date_str: str) -> datetime:
         """Parse date string and UTC offset safely into datetime."""
-        # Prepare current time with local timezone to use in case
-        # UTC offsets are wrong in Woffu.
+        # Prepare current time with local timezone
         # - We do it this way to take into account
         #   Daylight Saving Timezones (CET +1, CEST +2, for example).
         current_time = datetime.now(tz=self._localzone)
-
-        # WORKAROUND: Woffu stores incorrect UTC information
-        # when signing in from the Webapp, showing same local date
-        # on 'TrueDate' and 'UtcTime' but with no UTC offset (+0).
-        # - For the time being, we'll be using our local timezone
-        #   for any sign that comes with +0 in the UtcTime as we
-        #   suspect Woffu DB is running on a server with the same
-        #   local timezone as us.
-        utc_offset = f"{utc_offset.zfill(3)}00"
-        if utc_offset == UTC_OFFSET:
-            utc_offset = current_time.strftime("%z")
+        utc_offset = current_time.strftime("%z")
 
         date_timezoned = f"{date_str}{utc_offset}"
         try:

--- a/tests/test_woffu_api_client.py
+++ b/tests/test_woffu_api_client.py
@@ -1106,26 +1106,26 @@ class TestWoffuAPIStatusSign(BaseWoffuAPITest):
         self.assertIsNone(result)
         mock_post.assert_not_called()
 
-    @patch.object(WoffuAPIClient, "get")
-    def test_get_status_multiple_invalid_utc_formats(self, mock_get):
-        """get_status handles multiple invalid UtcTime formats."""
-        mock_get.return_value.status = 200
-        mock_get.return_value.json.return_value = [
-            {
-                "SignIn": True,
-                "TrueDate": "2025-09-12T12:00:00.000",
-                "UtcTime": "BAD1",
-            },
-            {
-                "SignIn": False,
-                "TrueDate": "2025-09-12T16:00:00.000",
-                "UtcTime": "BAD2",
-            },
-        ]
+    # @patch.object(WoffuAPIClient, "get")
+    # def test_get_status_multiple_invalid_utc_formats(self, mock_get):
+    #     """get_status handles multiple invalid UtcTime formats."""
+    #     mock_get.return_value.status = 200
+    #     mock_get.return_value.json.return_value = [
+    #         {
+    #             "SignIn": True,
+    #             "TrueDate": "2025-09-12T12:00:00.000",
+    #             "UtcTime": "BAD1",
+    #         },
+    #         {
+    #             "SignIn": False,
+    #             "TrueDate": "2025-09-12T16:00:00.000",
+    #             "UtcTime": "BAD2",
+    #         },
+    #     ]
 
-        total, running = self.client.get_status()
-        self.assertIsInstance(total, object)
-        self.assertFalse(running)
+    #     total, running = self.client.get_status()
+    #     self.assertIsInstance(total, object)
+    #     self.assertFalse(running)
 
     @patch.object(WoffuAPIClient, "get")
     def test_get_status_only_running_clock_last_sign_false(self, mock_get):
@@ -1145,21 +1145,6 @@ class TestWoffuAPIStatusSign(BaseWoffuAPITest):
         ]
         _, running = self.client.get_status(only_running_clock=True)
         self.assertFalse(running)
-
-    @patch.object(WoffuAPIClient, "get")
-    def test_get_status_invalid_utc_fallback_local(self, mock_get):
-        """Test invalid UtcTime parsing."""
-        mock_get.return_value.status = 200
-        mock_get.return_value.json.return_value = [
-            {
-                "SignIn": True,
-                "TrueDate": "2025-09-12T12:00:00.000",
-                "UtcTime": "INVALID",
-            },
-        ]
-        total, running = self.client.get_status()
-        self.assertTrue(running)
-        self.assertIsInstance(total, timedelta)
 
     @patch.object(WoffuAPIClient, "get")
     def test_get_status_invalid_utc_formats_mixed(self, mock_get):

